build_runner/lib/src/generate/phase.dart

```diff
@@ -30,9 +30,9 @@ abstract class BuildAction {
   /// Runs [builder] on [package] with [inputs] as primary inputs, excluding
   /// [excludes]. Glob syntax is supported for both [inputs] and [excludes].
   factory BuildAction(Builder builder, String package,
-      {List<String> inputs = const ['**'],
-      List<String> excludes = const [],
-      bool isOptional}) {
+      {List<String> inputs, List<String> excludes, bool isOptional}) {
+    inputs ??= const ['**'];
+    excludes ??= const [];
```

Comment:
This default should be handled by `InputSet` (since its an optional arg there and this is just forwarding it)

Reply:
Done.